cmd/minikube/cmd/start.go

```diff
@@ -172,6 +173,13 @@ func runStart(cmd *cobra.Command, args []string) {
 	console.OutStyle(console.Happy, "minikube %s on %s (%s)", version.GetVersion(), runtime.GOOS, runtime.GOARCH)
 	validateConfig()
 
+	currentUser, err := user.Current()
```

Comment:
could you log out the err if it no nil ? 
like "error getting the current user" (without failing) and there is an error getting the current user, skip the warning ... ( we dont want to fail minikube becaause we can't warn them about which user is best to run minikube